internal/resolve/resolve_test.go

```diff
@@ -12,13 +13,15 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const testPluginName = "test-plugin"
+
 func TestVersion(t *testing.T) {
 	testDataDir := t.TempDir()
 	currentDir := t.TempDir()
 	conf := config.Config{DataDir: testDataDir, DefaultToolVersionsFilename: ".tool-versions", ConfigFile: "testdata/asdfrc"}
-	_, err := repotest.InstallPlugin("dummy_plugin", conf.DataDir, "lua")
```

Comment:
Tests were failing for me locally because I actually have the lua plugin installed on my computer. With the change above to make the home directory version the default these tests started picking up my own personal config, so I changed to a test plugin name.